.chloggen/metric-index-from-annottaions.yaml

```
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, networkExplorer, operator, chart, other)
+component: other
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added support for rerouting metrics index with pod/namespace annotations
+# One or more tracking issues related to the change
+issues: []
```

```suggestion
issues: [1053]
```
failing changelog validation